rotkehlchen/inquirer.py

```diff
@@ -445,6 +448,22 @@ def find_usd_price(
             )
             return usd_price
 
+        # BSQ is a special asset that doesnt have oracle information but its custom API
+        if asset == A_BSQ:
+            try:
+                price_in_btc = get_bisq_market_price(asset)
+                btc_price = Inquirer().find_usd_price(A_BTC)
+                usd_price = Price(price_in_btc * btc_price)
+                Inquirer._cached_current_price[cache_key] = CachedPriceEntry(
+                    price=usd_price,
+                    time=ts_now(),
+                )
+                return usd_price
+            except (RemoteError, DeserializationError) as e:
+                msg = f'Could not find price for {asset}. {str(e)}'
```

Comment:
```suggestion
                msg = f'Could not find price for BSQ. {str(e)}'
```

Save ourselves an unneeded substituion. But also didn't you say it should be 100 sats? Isn't it better to use the "intended" price of 100 sats as fallback instead of `0`?